chaincerts_wallet/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub fn add(left: usize, right: usize) -> usize {
```

Comment:
@Odraxs Since this will provide a contract interface **certs_wallet** might be a proper name